.github/actions/mamba-env/action.yml

```diff
@@ -38,7 +38,7 @@ runs:
         activate-environment: ${{ inputs.activate-environment }}
         environment-file: ${{ inputs.environment-file }}
         python-version: ${{ inputs.python-version }}
-        channel-priority: strict
+        channel-priority: flexible
```

Comment:
why?

Reply:
To test pandas rc another channel is needed. In the case `strict` option isn't suitable.